api/envoy/api/v2/core/health_check.proto

```diff
@@ -212,6 +212,10 @@ message HealthCheck {
   // Specifies the path to the :ref:`health check event log <arch_overview_health_check_logging>`.
   // If empty, no event log will be written.
   string event_log_path = 17;
+
+  // If set to true, health check failure events will always be logged.
+  // The default to value for "Always_log_health_check_failures" is false.
```

Comment:
nit: "The default value is false."

Also, if set to false, the first failure would still be logged, right? Can you clarify?

Reply:
You're correct.  I've updated the comment.